components/raftstore/src/store/peer.rs

```diff
@@ -611,7 +611,7 @@ pub struct SnapshotRecoveryWaitApplySyncer {
 }
 
 impl SnapshotRecoveryWaitApplySyncer {
-    pub fn new(region_id: u64, sender: SyncSender<u64>) -> Self {
+    pub fn new(region_id: u64, sender: StdSender<u64>) -> Self {
```

Comment:
why change it?

Reply:
the thought is async channel may be better, however, according to the test, it looks does not matter, so to keep identical with wait apply PR, change back to SyncSender.